app/controllers/api/ims_imports_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::ImsImportsController < ApplicationController
+  include Concerns::CanvasImsccSupport
```

Comment:
It looks like this concern has respond_to :json in it too, so there's no need to do the respond_to in the create method.